api/lorawan.proto

```diff
@@ -242,6 +242,13 @@ message DataRate {
   };
 }
 
+
+message RequestInfo{
```

Comment:
Spacing

Reply:
What does "RequestInfo" mean on the V3 API level?

Reply:
It doesn't seem to be used. Is this ready to review?

Reply:
wtf? Where did this even come from? On it.

Reply:
This was a commit that was left over from an old rebase mistake. It's dropped now.